src/escolalms/sdk/components/Course/CourseProgramContent.tsx

```diff
@@ -129,6 +129,17 @@ export const CourseProgramContent: React.FC<{
 
       case TopicType.Pdf:
         return <PdfPlayer url={topic.topicable.url} />;
+
+      case TopicType.Scorm:
+        return (
+          <div className="scorm-wrapper">
+            <iframe
+              title={topic.topicable.value}
+              src={`${apiUrl}/api/scorm/play/${topic.topicable.uuid}`}
```

Comment:
@victazzz @dyfero maybe it's better if this `url` comes from API in the response